source/loader/CMakeLists.txt

```diff
@@ -15,6 +15,23 @@ add_ur_library(ur_loader
     ${CMAKE_CURRENT_BINARY_DIR}/UrLoaderVersion.rc
 )
 
+if (MSVC)
+    set(TARGET_LIBNAME ur_loader)
+    
+    set(LOADER_VERSION_SCRIPT ${CMAKE_CURRENT_BINARY_DIR}/ur_loader.def)
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/loader.def.in ${LOADER_VERSION_SCRIPT} @ONLY)
+    set_target_properties(ur_loader PROPERTIES
+        LINK_FLAGS "/DEF:${LOADER_VERSION_SCRIPT}"
+    )
+else()
```

Comment:
Done

Reply:
hm, doesn't seem to be working. Let's just do nothing on mac.

Reply:
Added an `.exp` file for macos. Builds now although I don't know if it actually works as planned :P

Reply:
Is the intended result that only the UR entry points are visible? If so, the macOS path doesn't seem to be doing that.

Reply:
We should include in the loader lib only symbols that we want to export, namely API functions. I don't have an access to macos locally so finding the way to solve this could take me a bit longer using macos available in CI. I can try finding a way to make it work on macos, we could also make it part of another PR and remove a special path for macOS from this PR completely, what do you think?

Reply:
I do have access to macOS. It probably makes sense to remove the macOS path from this PR and create a ticket to add this later, could you do that and assign it to me?

Reply:
Thanks, done